### PR TITLE
Loomio

### PR DESCRIPTION
**Pull request by mixmix**

### CODE REVIEW

File: people/viv-maidaborn.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - '@type': Relationship
+    type: relationshipTypes/stewardship
+    context: groups/loomio
+    roles:
+      - '@type': Role
+        type: roleTypes/steward
+        <<: *me 
+      - '@type': Role
+        type: roleTypes/stewardee
+        agent: people/viv-maidaborne
```
Comment (ahdinosaur) [inline people/viv-maidaborn.yml]: @mixmix Viv stewards herself?

Comment (mixmix) [inline people/viv-maidaborn.yml]: @ahdinosaur good catch
FIXED!

**Comment by mixmix** [comment]
keen to seen this live. 
lemme know when it is !

**Comment by ahdinosaur** [comment]
merged, should be up in [~15 minutes](https://github.com/holodex/app/blob/master/config/production.js#L6).

**Comment by ahdinosaur** [comment]
also thanks @mixmix for all your help, you're awesome. :smile_cat:

**Comment by ahdinosaur** [comment]
@mixmix: it's [live](http://holodex.enspiral.info/groups/loomio)!

[loomio-stewarding](http://holodex.enspiral.info/groups/loomio)

:tada: :confetti_ball: :dancer:

**Pull request merged**